Subscribers/SubscribeManager.php

```diff
@@ -32,6 +32,8 @@
      */
     public function __construct()
     {
+        parent::__construct();
```

Comment:
I think it is very critical to call the parten constructor here. If you take a look at `\Enlight_Class::__construct` I don't see any positive effect on it.